diff --git a/pyproject.toml b/pyproject.toml
index f23673e..d7037f8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 name="tarware"
 version="1.1.0"
 description="Task-Assignment Multi-Robot Warehouse environment for reinforcement learning"
-readme = {file = "README.txt", content-type = "text/markdown"}
+readme = {file = "README.md", content-type = "text/markdown"}
 authors = [
   {name = "Raul Steleac", email = "raulsteleac@ed.ac.uk"},
 ]
@@ -28,5 +28,8 @@ dependencies = [
 [project.optional-dependencies]
 test = ["pytest"]
 
+[tool.setuptools]
+packages = ["tarware", "tarware.spaces", "tarware.utils"]
+
 [project.urls]
 Repository  = "https://github.com/raulsteleac/task-assignment-robotic-warehouse.git"
\ No newline at end of file
